src/main/webapp/jsp/instructorFeedbackEdit.jsp

```diff
@@ -36,6 +40,7 @@
   </c:if>
    <br>
   <input type="hidden" id="num-questions" value="${fn:length(data.qnForms)}">
+  <input type="hidden" name="${FEEDBACK_SESSION_ENABLE_EDIT}" value="${data.isToBeLoadedInEditMode}">
```

Comment:
@whipermr5 When I place it in `feedbackSessionsForm.tag`, I get a server error (500) on the `instructorFeedbackSessionsPage`. I get this error because the `feedbackSessionsForm.tag` is reused here, but since there is no `isToBeLoadedInEditMode` parameter being sent to _that_ page (its only sent  to `instructorFeedbackEditPage`), the following div that I inserted gives the `500` error:

```HTML
<div name="${FEEDBACK_SESSION_ENABLE_EDIT}" data-value="${data.isToBeLoadedInEditMode}"></div>
```

Is there some way to avoid this error? Or would it be okay if I kept the current placement (ie: kept it in `instructorFeedbackEdit.jsp`) but changed the implementation from an `<input>` to a `<div></div>`